src/button/Button.tsx

```diff
@@ -54,6 +56,14 @@ function Button({
     "button--is-inactive": isButtonDisabled,
     "button--is-pending": shouldDisplaySpinner
   });
+  const spinnerContent = customSpinner || (
+    <Spinner
+      customClassName={classNames("button__spinner", {
+        [`${customClassName}__spinner`]: customClassName
```

Comment:
Let's not do this because it may lead to weird classes. For example, if we provide customClassName as `button--small button--primary` then this results in adding `button--small button--primary__spinner`. Let's just use `button__spinner` when we want to override the styles of the default Spinner

Reply:
Actually, this [PR](https://github.com/Hipo/react-ui-toolkit/pull/81) includes this solution. If it's OK, I can close this PR.